src/Aquifer.API/Endpoints/Bibles/Texts/Get/Response.cs

```diff
@@ -21,4 +21,5 @@ public class ResponseChapterVerses
 {
     public required int Number { get; set; }
     public required string Text { get; set; }
+    public string BaseMapping { get; set; } = null!;
```

Comment:
This name doesn't give a lot of context into the meaning of the data the property contains.  I suggest `public VerseReference? OriginalVerseReference { get; set; }` where `VerseReference` is:
```
public class VerseReference
{
    public required string BookName { get; set; }
    public required string BookCode { get; set; }
    public required int BookNumber { get; set; }
    public required int ChapterNumber { get; set; }
    public required int VerseNumber { get; set; }
}
```
(but only include the book info that you actually need)